src/player/PlayerComponent.cpp

```diff
@@ -1524,3 +1524,17 @@ QString PlayerComponent::videoInformation() const
   return infoStr;
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////
+void PlayerComponent::setPlaybackSpeed(double speed)
+{
+  mpv::qt::set_property(m_mpv, "speed", speed);
```

Comment:
Should this clamp to avoid negative double values?

Reply:
MPV will most likely clamp the value.